ios/RNSConvert.h

```diff
@@ -6,6 +6,13 @@ namespace react = facebook::react;
 
 @interface RNSConvert : NSObject
 
++ (NSNumber *)getFontSizePropValue:(int)value;
+
++ (UISemanticContentAttribute)getDirectionPropValue:(react::RNSScreenStackHeaderConfigDirection)direction;
+
++ (UINavigationItemBackButtonDisplayMode)getBackButtonDisplayModePropValue:
+    (react::RNSScreenStackHeaderConfigBackButtonDisplayMode)backButtonDisplayMode;
```

Comment:
Would suggest sticking to convention here:
```suggestion
+ (UINavigationItemBackButtonDisplayMode)UINavigationItemBackButtonDisplayModeFromCppEquivalent:
    (react::RNSScreenStackHeaderConfigBackButtonDisplayMode)backButtonDisplayMode;
```

Reply:
And it would be great to move these in separate PR. But this time it is fine I guess.